src/vaadin-crud.html

```diff
@@ -415,6 +426,7 @@ <h3 slot="header">[[__editorHeader]]</h3>
           this.__editListener = e => this.__onCrudGridEdit(e);
           this.__changeListener = e => this.__onFormChanges(e);
           this.__gridSizeListener = this.__onGridSizeChanges.bind(this);
+          this.__clickRowToEditListener = this.__clickRowToEditListener.bind(this);
```

Comment:
Not sure, but I am slightly afraid that there could be a case when it won't remove the listener (cause of the same name used). Haven't tested though.

We can use the same approach as in grid: https://github.com/vaadin/vaadin-grid/blob/51240f4fbcff7ad0f527add14e34ed3cb5f57875/src/vaadin-grid-selection-column.html#L148
just to be on safe side. (use `_bound` prefix)

Reply:
Done.